Test/Pester5.Tests.ps1

```diff
@@ -297,9 +173,32 @@ Describe 'Failing' {
     }
 }
 '@
-        WhenPesterTaskIsInvoked -AsJob -ErrorAction SilentlyContinue
-        ThenPesterShouldHaveRun -FailureCount 2 -PassingCount 0
-        # ThenTestShouldFail -failureMessage 'Pester tests failed'
+        WhenPesterTaskIsInvoked -AsJob -WithArgument @{

```

Comment:
I think the only property you need to pass to Pester is `Run.Path` since you're only testing that the task fails.